simplify signature dependency so that it translates to something Debian understands

diff --git a/Cargo.toml b/Cargo.toml
index c0a3c8c..d120db5 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -62,7 +62,7 @@ version = "0.10"
 default-features = false
 
 [dependencies.signature]
-version = "2.0, <2.3"
+version = "2"
 features = [
     "alloc",
     "digest",
